include/aspect/initial_temperature/S40RTS_perturbation.h

```diff
@@ -186,6 +189,7 @@ namespace aspect
          */
         unsigned int vs_to_density_index;
 
+        template <int dim2> friend class PatchOnS40RTS;
```

Comment:
Is there a reason this has to be `dim2`? Does `dim` not work?

Reply:
Yes, dim was throwing an error, so Wolfgang had me change it to dim2